h/features/models.py

```diff
@@ -35,6 +35,23 @@
 #
 FEATURES_PENDING_REMOVAL = {}
 
+FEATURE_FEATURECOHORT_TABLE = sa.Table(
+    'feature_featurecohort', Base.metadata,
+    sa.Column('id',
+              sa.Integer(),
+              nullable=False),
+    sa.Column('feature_id',
+              sa.Integer(),
+              nullable=False),
+    sa.Column('cohort_id',
+              sa.Integer(),
+              nullable=False),
+    sa.PrimaryKeyConstraint('id'),
+    sa.ForeignKeyConstraint(['cohort_id'], ['featurecohort.id']),
```

Comment:
Again, let's be consistent in how we do these things and use the same pattern as you used for the user join table below.